cmd/zpool/zpool_main.c

```diff
@@ -519,7 +518,7 @@ print_vdev_prop_cb(int prop, void *cb)
  * that command.  Otherwise, iterate over the entire command table and display
  * a complete usage message.
  */
-static void
+static void __attribute__((noreturn))
```

Comment:
Are we doing `_Noreturn` instead now?

Reply:
We are, I'm just deep in ZTS mines and haven't rebased this since we merged that

Reply:
Speaking of the ZTS mines, it would be good to verify there are some negative tests which verify the stricter "only accept" changes are working correctly.

Reply:
Rebased, fixed the `__attribute__((noreturn))`s, and added validation tests (top commit).